data/strings/sound.txt

```diff
@@ -399,7 +471,8 @@
     da = Afkørsel.
     de = Nehmen Sie die Ausfahrt.
     el = Εξοδος.
-    es = Salida.
+    es = Salga
```

Comment:
Are you sure that no dot is needed here and below?

Reply:
It doesn't really matter for something like this because it's either always at the end of a sentence anyway or my logic removes terminal dots when it's in the middle of a sentence.

Before with a dot:

`In 500 feet, Exit.`
`Exit.`

Now with or without a dot:

`In 500 feet, Exit onto Main Street`
`Exit onto Main Street`

TTS is almost always smart enough to still sound authoritative when given a sentence fragment without terminal punctuation, so honestly I'd prefer having no periods at all but it really doesn't matter. I've re-added it here.